scripts/01-test

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# Scripts in this directory are run during the build process.
+# each script will be uploaded to /tmp on your build droplet, 
+# given execute permissions and run.  The cleanup process will
+# remove the scripts from your build system after they have run
+# if you use the build_image task.
+#
+echo "HELLO WORLD"
```

Comment:
We don't necessarily need this test command, right?

Reply:
Nope!